src/composables/useTresContextProvider/index.ts

```diff
@@ -45,6 +45,20 @@ export interface PerformanceState {
   }
 }
 
+interface TresEventManager {
```

Comment:
(Non-blocking)

At first glance, it seems like this should be in `src/componsables/useTresEventManager/index.ts`.